src/Components/Book.js

```diff
@@ -10,23 +10,43 @@ const Book = (props) => {
   const dispatch = useDispatch();
 
   return (
-    <article>
-      <h2>{title}</h2>
-      <h3>{author}</h3>
-      <p>{category}</p>
-      <ul className={styles.book_btn_container}>
-        <li>
-          <button type="button">Comments</button>
-        </li>
-        <li>
-          <button type="button" onClick={() => dispatch(deleteBook(id))}>
-            Remove
-          </button>
-        </li>
-        <li>
-          <button type="button">Edit</button>
-        </li>
-      </ul>
+    <article className={styles.article}>
```

Comment:
- [ ] It would be better if we increase the width of the bookList and the bookForm to match the given design. Here is a screenshot of our design and the required to have a more clearer view
Our Design
![Screenshot from 2023-05-18 13-56-48](https://github.com/bilalrajput09/Bookstore-react/assets/46388113/97e36d2a-ec27-440c-86c9-74e694bdde97)

Required Design
![Screenshot from 2023-05-18 13-57-39](https://github.com/bilalrajput09/Bookstore-react/assets/46388113/c0e683eb-b3e8-47ce-bb51-c49f42d93860)
**NB**: Here we can see the difference clearly, considering the text size, weight, etc